
epoch:  0 , example:  1  current loss =  2.845151901245117
epoch:  0 , example:  2  current loss =  2.5612759590148926
epoch:  0 , example:  3  current loss =  2.40611998240153
epoch:  1 , example:  4  current loss =  2.2652744352817535
epoch:  1 , example:  5  current loss =  2.1689504384994507
Traceback (most recent call last):
  File "/media/jonas/B41ED7D91ED792AA/Arbeit_und_Studium/Kognitionswissenschaft/Semester_5/masterarbeit#/data_Code/code/completeTrainScript.py", line 79, in <module>
    functions.trainLoop(dTrain, model, False,"transformerPatches", 0.0001, 0.01, 0.00001, 2, dValidate, 10, True, device)
  File "/media/jonas/B41ED7D91ED792AA/Arbeit_und_Studium/Kognitionswissenschaft/Semester_5/masterarbeit#/data_Code/code/functions.py", line 555, in trainLoop
    runningLossLatentSpace[trainCounter] = meanRunningLossLatentSpace
IndexError: index 6 is out of bounds for axis 0 with size 6
Traceback (most recent call last):
  File "/media/jonas/B41ED7D91ED792AA/Arbeit_und_Studium/Kognitionswissenschaft/Semester_5/masterarbeit#/data_Code/code/completeTrainScript.py", line 79, in <module>
    functions.trainLoop(dTrain, model, False,"transformerPatches", 0.0001, 0.01, 0.00001, 2, dValidate, 10, True, device)
  File "/media/jonas/B41ED7D91ED792AA/Arbeit_und_Studium/Kognitionswissenschaft/Semester_5/masterarbeit#/data_Code/code/functions.py", line 555, in trainLoop
    runningLossLatentSpace[trainCounter] = meanRunningLossLatentSpace
IndexError: index 6 is out of bounds for axis 0 with size 6